src/ai/backend/manager/models/agent.py

```diff
@@ -88,6 +88,7 @@ class AgentStatus(enum.Enum):
     sa.Column("version", sa.String(length=64), nullable=False),
     sa.Column("architecture", sa.String(length=32), nullable=False),
     sa.Column("compute_plugins", pgsql.JSONB(), nullable=False, default={}),
+    sa.Column("auto_terminate", sa.Boolean(), nullable=False, server_default=true(), default=False),
```

Comment:
I think the column name is too vague. How is "auto_terminate" different from "batch-type session"?
The naming should include hints for abusing.